Source/WebCore/Modules/async-clipboard/ClipboardItemBindingsDataSource.cpp

```diff
@@ -156,7 +156,7 @@ void ClipboardItemBindingsDataSource::collectDataForWriting(Clipboard& destinati
                 return;
 
             Ref itemTypeLoader { *weakItemTypeLoader };
-#if !COMPILER(MSVC)
+#if !COMPILER(MSVC) || COMPILER(CLANG)
```

Comment:
It's kind of difficult to remember at a glance that COMPILER(MSVC) and COMPILER(CLANG) can both be true. If we don't want to add a new COMPILER, I think `!(COMPILER(MSVC) && !COMPILER(CLANG))` would at least make it clearer that it's not a typo.

Reply:
Fixed.
I might have taken a wrong decision. https://bugs.webkit.org/show_bug.cgi?id=171618#c10
Anyway, I hope we will remove MSVC support soon.